### PR TITLE
Finish v2 documentation cleaning

### PR DESCRIPTION
**Pull request by gyuho**
Second commit is:

```
Remove:
1. auth_api.md
2. docker_guide.md
3. faq.md
4. implementation-faq.md
5. internal-protocol-versioning.md
```

### CODE REVIEW

File: Documentation/op-guide/runtime-configuration.md [hunk 1]
```diff
@@ -177,10 +177,10 @@ It is recommended to enable this option. However, it is disabled by default beca
 [cluster-reconf]: #cluster-reconfiguration-operations
 [conf-adv-peer]: configuration.md#-initial-advertise-peer-urls
 [conf-name]: configuration.md#-name
-[disaster recovery]: admin_guide.md#disaster-recovery
-[fault tolerance table]: admin_guide.md#fault-tolerance-table
+[disaster recovery]: ../v2/admin_guide.md#disaster-recovery
```
Comment (xiang90) [inline Documentation/op-guide/runtime-configuration.md]: probably link to v3 version at recover.md in the same directory?

File: Documentation/op-guide/runtime-reconf-design.md [hunk 2]
```diff
@@ -47,4 +47,4 @@ It seems that using public discovery service is a convenient way to do runtime r
 If you want to have a discovery service that supports runtime reconfiguration, the best choice is to build your private one.
 
 [add-member]: runtime-configuration.md#add-a-new-member
-[disaster-recovery]: admin_guide.md#disaster-recovery
+[disaster-recovery]: ../v2/admin_guide.md#disaster-recovery
```
Comment (xiang90) [inline Documentation/op-guide/runtime-reconf-design.md]: same here.

Comment (gyuho) [inline Documentation/op-guide/runtime-reconf-design.md]: just fixed.

File: Documentation/op-guide/runtime-configuration.md (hunk 1 above)
Comment (gyuho) [inline Documentation/op-guide/runtime-configuration.md]: just fixed.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**